docusaurus/docs/how-to-guides/app-plugins/error-handling-in-app-plugins.md

```diff
@@ -0,0 +1,79 @@
+---
+id: error-handling-in-app-plugins
+title: Error handling in app plugins
```

Comment:
Not sure if we need _in app plugins_ in the sidebar? The additional context is provided by the folder and it just looks odd (to me at least). Could override with `sidebar_label: Error handling` and `sidebar_label: Include dashboards` for these panels

<img width="287" alt="Screenshot 2024-07-01 at 11 30 16" src="https://github.com/grafana/plugin-tools/assets/19860021/10fc30e0-59a3-42cb-a652-227325bf87cc">
<img width="295" alt="Screenshot 2024-07-01 at 11 33 19" src="https://github.com/grafana/plugin-tools/assets/19860021/79d3d14d-c4aa-4086-83b8-b2ec1c2292cb">

Reply:
_in X plugin_ doesn't appear in the datasources PR by comparison

Reply:
I'm going to resolve this for now as it is the same with panels. We can revisit in the datasources PR or tidy up for consistency in a later PR